bridge/engine/UntypedEngineObject.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+namespace bridge
+{
+
+
+struct UntypedEngineObject
```

Comment:
I do not like the term untyped. Why not 
GenericEngineObject

Reply:
I think this class should be removed completely and use template PendingJobsAsyncWaitTask